src/Compilers/CSharp/Test/IOperation/IOperation/IOperationTests_IForLoopStatement.cs

```diff
@@ -2690,6 +2690,25 @@ static void Main(string[] args)
         }
     }
 }
+";
+            var tree = GetOperationTreeForTest<ForStatementSyntax>(source);
```

Comment:
no tree produced now since hte start/end of hte span is not validly containing the for-statement.  it worked before because parsing was so thrown off it through the for loop was terminated.